spec/src/main/asciidoc/microprofile-lra-spec.adoc

```diff
@@ -910,7 +907,15 @@ is still in progress using one of the following options:
   it exists, and if there is no such method it will reinvoke the `@Compensate`
   method). Please refer to the section
   <<reactive-support,about reactive support>> for more details.
-- A JAX-RS method can return a `202 Accepted` HTTP status code
+- A JAX-RS method can return a `202 Accepted` HTTP status code.
+  If there is no `@Status` method then the response MAY provide
+  a status URL in the `HTTP Location` header field so that the
+  implementation can discover the final outcome.
```

Comment:
Ok, During the discussions I always had the impression that the location Header would be made available in the case when there IS @Status method available.

In this scenario, the Location Header becomes an alternative for the @Status method and reinvoking the Compensate method and makes sense.